scripts/drivers/doppler.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
+# shellcheck disable=SC2034
+_DRIVER_REGEX='!doppler [a-z0-9_-]*\#.*\#[A-Z0-9_]*'
+
+# shellcheck source=scripts//drivers/_custom.sh
+. "${SCRIPT_DIR}/drivers/_custom.sh"
+
+_doppler() {
+    # shellcheck disable=SC2086
+    set -- ${SECRET_DRIVER_ARGS} "$@"
+    doppler "$@"
+}
+
+_custom_driver_get_secret() {
+    _type=$1
+    _SECRET=$2
+
+    # Tokenize
+    IFS="#"
+    # shellcheck disable=SC2086
+    set -- $_SECRET
+    project=$1
+    config=$2
+    secret=$3
+
+    _doppler secrets get -p "${project}" -c "${config}" "${secret}" --plain || exit 1
```

Comment:
Does provide doppler a good enough error message?

Otherwise, don't fail silent here.

```suggestion
    if ! _doppler secrets get -p "${project}" -c "${config}" "${secret}" --plain; then
        echo "Error while get secret from doppler!" >&2
        echo doppler secrets get -p "${project}" -c "${config}" "${secret}" --plain >&2
        exit 1
    fi
```

Reply:
Yes it does ! 

```bash
$ helm secrets -d ./helm-secrets/scripts/drivers/doppler.sh view helm-test/values.yaml
Doppler Error: Could not find requested secret: DEMO_TOKEN
Error: plugin "secrets" exited with error
```